azure-pipelines-richnav.yml

```
@@ -16,7 +16,7 @@ stages:
         enableRichCodeNavigation: true
         richCodeNavigationEnvironment: 'production'
         pool:
-          name: NetCore1ESPool-Svc-Public
+          name: NetCore-Svc-Public
           demands: ImageOverride -equals Build.Windows.Amd64.VS2022.Pre.Open
```

If we're servicing rc1 branch, then we should use this opportunity to update the image names as well - similar to https://github.com/dotnet/winforms/pull/7698
```suggestion
          demands: ImageOverride -equals windows.vs2022preview.amd64.open
```